### ISSUE TITLE
the kernel process exited (-11)

### ISSUE THREAD

**Issue by prtome** [pyzo/pyzo#576]
on MacOS 10.9.5 - Installed Pyzo and Anaconda, latest version, new installs.
This message appears in the console at startup 

       the kernel process exited (-11)

And of course nothing works. Anaconda Python works perfectly when called for the Terminal

Is this a known issue (could not find ) . Thanks

**Comment by almarklein** [comment]
Check your shell configuraration and make sure that the `exe` points to the correct Python executable.

**Comment by prtome** [comment]
Yes it does, I checked that: the shell provides first the standard information about the version of Python installed (using Anaconda Python 3). I confirmed that when I got the first Pyzo start page.
There is no other Python 3 installation on that machine 

        Python 3.7.0 (default, Jun 28 2018, 07:39:16) on darwin (64 bits).
        This is the Pyzo interpreter with integrated event loop for PYQT5.
        Type 'help' for help, type '?' for a list of *magic* commands.


The Kernel message comes after that. As I said I checked the same Python version  using the Terminal, and it works perfectly.
Same installation pf Pyzo and anaconda on a Mac with 10.14 works without problems, so i am wondering if it is a problem on 10.9 - That Mac is only used for school work, there are no games or other stuff not school related.

**Comment by almarklein** [comment]
Right, so Python correctly starts up, but then somehow crashes. Could you start by (again in the shell config) disable GUI integration, to see whether PyQt5 us the source of trouble?

**Comment by bdieterm** [comment]
No response from the issue creator for more than 7 years.
